cypress/e2e/gui/messenger.cy.js

```diff
@@ -0,0 +1,50 @@
+describe('EngageSphere', () => {
+  beforeEach(() => {
+    cy.visit('/')
+  })
+
+  it('opens and closes the messenger', () => {
+    cy.get('[aria-label*="messenger"]').click()
+    cy.contains('h2', 'How can we help you?').should('be.visible')
+
+    cy.get('[aria-label*="messenger"]').click()
+    cy.contains('h2', 'How can we help you?').should('not.exist')
+  })
+
+  it('makes sure all messenger\'s fields are mandatory and the first one is focused', () => {
+    cy.get('[aria-label*="messenger"]').click()
```

Comment:
```suggestion
    cy.get('[aria-label*="messenger"]').click()

```

É importante separar ação de resultado esperado para que ao "bater o olho" no teste seja possível entender o que é o que.